corehq/apps/export/models/new.py

```diff
@@ -1075,10 +1080,11 @@ def _get_app_build_ids_to_process(domain, last_app_versions):
         return map(lambda app_build_version: app_build_version.build_id, app_build_verions)
 
     @staticmethod
-    def generate_schema_from_builds(domain, case_type, force_rebuild=False):
+    def generate_schema_from_builds(domain, app_id, case_type, force_rebuild=False):
```

Comment:
makes sense. I don't feel too strongly about it. I think I've always thought of the app selection in the from/case type selection modal as more of a sort of filter on the forms/case types than as something that will affect the export, but that may not be how others interpret it.